app/src/androidTest/java/com/reach5/identity/sdk/demo/SignupSuccessTest.kt

```diff
@@ -0,0 +1,113 @@
+package com.reach5.identity.sdk.demo
+
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.Espresso.pressBack
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.*
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class SignupSuccessTest {
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
+    private fun anEmail(): String = UUID.randomUUID().let { uuid -> "$uuid@testaccount.io" }
+    val newEmail = anEmail()
+
+    @Test
+    fun signupSuccessTest() {
+        val appCompatEditText = onView(
+            allOf(
+                withId(R.id.email),
+                childAtPosition(
+                    childAtPosition(
+                        withClassName(`is`("android.widget.LinearLayout")),
+                        0
+                    ),
+                    2
+                )
+            )
+        )
+
+        appCompatEditText.perform(
+            scrollTo(),
+            replaceText(newEmail),
+            closeSoftKeyboard()
+        )
+
+        val appCompatEditText2 = onView(
+            allOf(
+                withId(R.id.password),
+                childAtPosition(
+                    childAtPosition(
+                        withClassName(`is`("android.widget.LinearLayout")),
+                        0
+                    ),
+                    4
+                )
+            )
+        )
+        appCompatEditText2.perform(scrollTo(), replaceText("Test1806@"), closeSoftKeyboard())
```

Comment:
A quoi sert `Test1806@` ?

Reply:
c'est le password du nouveau compte

Reply:
Can we please name the value somewhere? `replaceText("Test1806@")` isn't very clear.